scripts/build.sh

```diff
@@ -211,6 +212,16 @@ then
 
     if [[ IS_TEST_MODE -eq 0 ]]
     then
+        if [ -z "$AMPLITUDE_API_KEY" ]
```

Comment:
have you tested this by trying to do a release with the `-d` flag and the `-f` flag?

Reply:
Yup! Both exhibited the desired behaviors.